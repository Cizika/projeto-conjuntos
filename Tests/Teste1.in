1 1 8
1 1 7
1 2 2
1 2 4
1 2 7
1 2 10
1 2 14
3 7
2 2 7
3 7
4 1 2
5 1 2
0