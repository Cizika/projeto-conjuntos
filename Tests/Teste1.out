--------------------------------------------------
CONJUNTOS MATEMÁTICOS
1 - Insere um elemento no conjunto desejado
2 - Remove um elemento do conjunto desejado
3 - Verifica se o elemento pertence aos conjuntos
4 - União entre conjuntos
5 - Intersecção entre conjuntos
0 - Finalizar Programa
--------------------------------------------------
Conjunto 1: {8 }
Conjunto 2: {}
Conjunto 3: {}
--------------------------------------------------
Conjunto 1: {7 8 }
Conjunto 2: {}
Conjunto 3: {}
--------------------------------------------------
Conjunto 1: {7 8 }
Conjunto 2: {2 }
Conjunto 3: {}
--------------------------------------------------
Conjunto 1: {7 8 }
Conjunto 2: {2 4 }
Conjunto 3: {}
--------------------------------------------------
Conjunto 1: {7 8 }
Conjunto 2: {2 4 7 }
Conjunto 3: {}
--------------------------------------------------
Conjunto 1: {7 8 }
Conjunto 2: {2 4 7 10 }
Conjunto 3: {}
--------------------------------------------------
Conjunto 1: {7 8 }
Conjunto 2: {2 4 7 10 14 }
Conjunto 3: {}
--------------------------------------------------
O elemento 7 pertence ao conjunto 1
O elemento 7 pertence ao conjunto 2
O elemento 7 NÃO pertence ao conjunto 3
--------------------------------------------------
Conjunto 1: {7 8 }
Conjunto 2: {2 4 10 14 }
Conjunto 3: {}
--------------------------------------------------
O elemento 7 pertence ao conjunto 1
O elemento 7 NÃO pertence ao conjunto 2
O elemento 7 NÃO pertence ao conjunto 3
--------------------------------------------------
União(1, 2): {2 4 7 8 10 14 }
--------------------------------------------------
Intersecção(1, 2): {}
--------------------------------------------------
Finalizando Programa
--------------------------------------------------
